manifests/monitor/monitor-configMap.yaml

```diff
@@ -0,0 +1,190 @@
+        replacement: $1.$2-tidb-peer:$3
+        action: replace
+      - source_labels: [__meta_kubernetes_namespace]
+        action: replace
+        target_label: kubernetes_namespace
+      - source_labels: [__meta_kubernetes_pod_node_name]
+        action: replace
+        target_label: kubernetes_node
+      - source_labels: [__meta_kubernetes_pod_ip]
+        action: replace
+        target_label: kubernetes_pod_ip
+      - source_labels: [__meta_kubernetes_pod_name]
+        action: replace
+        target_label: instance
+      - source_labels: [__meta_kubernetes_pod_label_app_kubernetes_io_instance]
+        action: replace
+        target_label: cluster
+    - job_name: 'tikv'
+      scrape_interval: 15s
+      honor_labels: true
+      kubernetes_sd_configs:
+      - role: pod
+        namespaces:
+          names:
+          - default
+      scheme: http
+      tls_config:
+        insecure_skip_verify: true
+      # TiKV doesn't support scheme https for now.
+      # And we should fix it after TiKV fix this issue: https://github.com/tikv/tikv/issues/5340
+      #
+      # scheme: http
+      # tls_config:
+      #   insecure_skip_verify: true
+      #
+      relabel_configs:
+      - source_labels: [__meta_kubernetes_pod_label_app_kubernetes_io_instance]
+        action: keep
+        regex: test
+      - source_labels: [__meta_kubernetes_pod_label_app_kubernetes_io_component]
+        action: keep
+        regex: tikv
+      - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_scrape]
+        action: keep
+        regex: true
+      - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_path]
+        action: replace
+        target_label: __metrics_path__
+        regex: (.+)
+      - source_labels: [__meta_kubernetes_pod_name, __meta_kubernetes_pod_label_app_kubernetes_io_instance,
+          __meta_kubernetes_pod_annotation_prometheus_io_port]
+        regex: (.+);(.+);(.+)
+        target_label: __address__
+        replacement: $1.$2-tikv-peer:$3
+        action: replace
+      - source_labels: [__meta_kubernetes_namespace]
+        action: replace
+        target_label: kubernetes_namespace
+      - source_labels: [__meta_kubernetes_pod_node_name]
+        action: replace
+        target_label: kubernetes_node
+      - source_labels: [__meta_kubernetes_pod_ip]
+        action: replace
+        target_label: kubernetes_pod_ip
+      - source_labels: [__meta_kubernetes_pod_name]
+        action: replace
+        target_label: instance
+      - source_labels: [__meta_kubernetes_pod_label_app_kubernetes_io_instance]
+        action: replace
+        target_label: cluster
+    rule_files:
+      - '/prometheus-rules/rules/*.rules.yml'
+
+  dashboard-config: |-
+    {
+        "apiVersion": 1,
+        "providers": [
+            {
+                "folder": "",
+                "name": "0",
+                "options": {
+                    "path": "/grafana-dashboard-definitions/tidb"
```

Comment:
How to support additional dashboards for tidb monitor? We should support this as well

Reply:
It seems we could also export grafana config in the configmap.

Reply:
If we support custom dashboards by configMap , we will need to  append extra configMap to volumes and VolumeMounts . reference https://github.com/pingcap/tidb-operator/blob/7ccb528917432e2ba9f742a0ed01c4a8e47e71c4/pkg/monitor/monitor/util.go#L312. 
And I think users can dynamically add dashboard in Grafana.So, this feature is  not important.